vendor/vendor.json

```diff
@@ -6,6 +6,44 @@
 			"path": "context",
 			"revision": ""
 		},
+		{
```

Comment:
it would be nice if we could move to new secboot, etc. in a separate PR and then just use the new stuff in this PR, it would be easier to review and I would feel more comfortable that we aren't breaking something in this PR with the new secboot changes

Reply:
yeah, i'm not sure who is working on revving secboot and friends in snapd master. I guess I can attempt to start a fresh PR that does just that.

Reply:
Well I can tell you I'm not working on it, but @mvo5 is the best person to ask/coordinate with